File: SO1/EX1/main.go
package main

import (
	"fmt"
	"time"
)

func sayHello(name string) {
	fmt.Printf("Hello word %s!", name)
}

func main() {
	sayHello("Paulo")

	go sayHello("Paulo Vinicius")

	go func(msg string) {
		fmt.Println(msg)
	}("next")

	time.Sleep(time.Second)
	fmt.Println("done")
}

File: SO1/EX1/main.py
import _thread
import time

def say_hello(name: str, delay: int):
  count = 0
  while count < 5:
    time.sleep(delay)
    count += 1
    print ("%s: %s" % ( name, time.ctime(time.time()) ))

try:
   _thread.start_new_thread(say_hello, ("Paulo", 2) )
   _thread.start_new_thread(say_hello, ("Vinicius", 4) )
except:
   print ("Error: unable to start thread")

while 1:
   pass